### ISSUE TITLE
Determine Initial Value

### ISSUE THREAD

**Issue by jakedohm** [simplygoodwork/simplygoodforms#4]
Headed to bed, but I didn't want to forget this. WIll report back with more coherent thoughts probably tomorrow.

```js
// TEMP: This needs to be replaced with _actual_ logic that decides the initial value.
// If there is one passed in, it should obviously be set to that (unless there's a weird edge case).
// If there's _not_ a values passed in, we should probably set it to an "empty" value, depending on its type.
// We should revisit this, and see what other libraries do.

const initialValue = this.value ? this.value : ''
```

**Comment by chasegiunta** [comment]
Yeah, not sure what's best. Setting it to `null` could help in determining if the field has been touched or not (granted we're v-modeling `value`). So on a required input, don't report an error if value == `null` because it hasn't been touched yet.

That seems like a smaller, less thorough, interim solution until we have crazy states for each input `dirty, clean, touched, etc.`

But also completely inapplicable if we're just reading `value` prop on mount and setting it to something else for v-modeling.

**Comment by chasegiunta** [comment]
Looks like my initial implementation was to set this to `null` by default (unless a value is provided of course)

Let's run with that unless/until we run into an issue.

**Comment by jakedohm** [comment]
Works for me!
